### PR TITLE
Add tarball

### PR DESCRIPTION
**Pull request by ceyhun**
Committed `.tgz` file that's created after running `npm install` and `npm pack`

### CODE REVIEW

**Review by fluiddot — COMMENTED** [review]
I see on other forks we included the suffix `-1` in the version, shouldn't we do the same here?

**Potential package version:** `0.8.8-wp-1`

**Comment by ceyhun** [comment]
Yeah the first fork releases I had created didn't have the `-1` and we started to follow the `-N` convention from the second fork releases. This one didn't have a second release yet, so I think we should add the `-N` starting from the next release as well.

**Comment by fluiddot** [comment]
> Yeah the first fork releases I had created didn't have the `-1` and we started to follow the `-N` convention from the second fork releases. This one didn't have a second release yet, so I think we should add the `-N` starting from the next release as well.

I've just added the `-N` in [this commit](https://github.com/wordpress-mobile/react-native-keyboard-aware-scroll-view/pull/15/commits/3a62f1b1d8827456f73cd62ae05a6c50ca6d8abd).

**Comment by ceyhun** [comment]
Just to note though this repo doesn't have a `0.8.8-wp-1` release yet.
Also we might need to re-pack the tarball with `npm pack` so it has the correct `package.json` and filename.

**Comment by fluiddot** [comment]
> Just to note though this repo doesn't have a `0.8.8-wp-1` release yet.
> Also we might need to re-pack the tarball with `npm pack` so it has the correct `package.json` and filename.

Done!

I also committed the `package-lock.json` file, although I noticed that on the other forks we didn't 🤔 , should we?

**Comment by ceyhun** [comment]
I'm not sure if it's taken into consideration during Gutenberg's `npm install`, maybe it could cause multiple versions of a certain package being installed to match the ones `package-lock.json` or maybe not :)

**Comment by fluiddot** [comment]
> I'm not sure if it's taken into consideration during Gutenberg's `npm install`, maybe it could cause multiple versions of a certain package being installed to match the ones `package-lock.json` or maybe not :)

I see, I'll revert the change then. In this case, I think we should note that changes in the `package-lock.json` file shouldn't be committed, wdyt?

**Comment by ceyhun** [comment]
Yeah, I think that's closer to the current behaviour on Gutenberg CI, but again I'm not sure, it could also be just fine 🙂

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-keyboard-aware-scroll-view",
-  "version": "0.8.8-wp",
+  "version": "0.8.8-wp-1",
```
Comment (fluiddot) [inline package.json:3]: This version doesn't have a Git tag so we should create it once this gets merged.

**Review by fluiddot — APPROVED** [review]

**Pull request merged**